app/views/lookbook/previews/preview.html.erb

```diff
@@ -1,9 +1,5 @@
 <% if @render_args[:component] %>
-  <% if defined?(Phlex::SGML) && @render_args[:component].is_a?(Phlex::SGML) %>
-    <%= raw(@render_args[:component].call(view_context: self, &@render_args[:block])) %>
-  <% else %>
-    <%= render(@render_args[:component], @render_args[:args], &@render_args[:block]) %>
-  <% end %>
+  <%= render(@render_args[:component], @render_args[:args], &@render_args[:block]) %>
```

Comment:
I'm not sure why this was `raw + call`, but changing to `render component, args, block` worked on specs and on my project. 
Using `render` Rails calls `component.render_in(self, &block)`. Ref: https://github.com/rails/rails/blob/main/actionview/lib/action_view/helpers/rendering_helper.rb#L42

Reply:
@stephannv the discussion in #400 should help explain the rationale for this, ugly as it is!